data/mods/No_Explosive_Zombies/modinfo.json

```diff
@@ -0,0 +1,17 @@
+[
+{
+    "type": "MOD_INFO",
+    "mod-type": "SUPPLEMENTAL",
+    "ident": "No_Explosive_Zombies",
+    "name": "No Explosive Zombies",
+    "description": "Removes all explosion-based zombies from the game.",
```

Comment:
They don't deal any damage.
This is supposed to be less of a thematic blacklist and more of a balancing one. Boomers and bloateds don't fit among deadly zeds like grenadiers and future kamikaze zeds.